src/NServiceBus.Core.Tests/Sagas/ComponentTests/SagaWithComplexTypeEntity.cs

```diff
@@ -0,0 +1,10 @@
+namespace NServiceBus.Persistence.ComponentTests
+{
+    using System.Collections.Generic;
+
+    public class SagaWithComplexTypeEntity : ContainSagaData
+    {
+        public string CorrelationProperty { get; set; }
+        public List<int> Ints { get; set; }
```

Comment:
How much value is this adding? I guess each persister would need to add more comprehensive tests suits that exercises their way of serializing anyway? (eg. mappings for NH, Json.net for SqlPersister, DataContracts.Json for the learning one)